index.html

```diff
@@ -165,6 +165,30 @@ <h4 class="connect">LET'S CONNECT</h4>
 </ul>
 </Section>
 
-  
+<div class="container1">
+  <h2 class="contact-us-heading">Contact me</h2>
+  <p>If you have an application you are interested
+    in developing, a feature that you need built or a project that needs coding.
+    I’d love to help with it!</p>
+<div class="form">
+  <form action="#" method="post">
+    <label for="name">
+      <input type="text" id="name" name="name" placeholder="Enter your name" maxlength="30" required>
+    </label>
+    <label for="email">
+      <input type="email" id="email" name="email" placeholder="Enter your email" required>
+    </label>
+    <label for="message">
+      <textarea id="message" name="message" cols="30" rows="5" maxlength="500" placeholder="Enter your message here..." required></textarea>
+    </label>
+    <button type="submit" class="contact-btn">Get in touch</button>
+  </form>
+</div>
```

Comment:
- [ ] It is required to stick to the design as close as possible. This simply means that, you're expected to produce a close replica of the design. At this moment, your design does not look like how it is on the Figma design. Kindly look into the contact me section on the Figma design and work your design alongside that design.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/235907502-11bf9c33-3aba-4b71-94f2-feb6493358f7.png)

### Figma Design
![temp-design](https://user-images.githubusercontent.com/26597432/235907523-ca3f757d-0f76-477e-b312-dc2e381061e7.png)